dry-types.gemspec

```diff
@@ -33,9 +33,10 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'dry-configurable', '~> 0.1'
   spec.add_runtime_dependency 'dry-logic', '~> 0.3', '>= 0.3.0'
   spec.add_runtime_dependency 'inflecto', '~> 0.0.0', '>= 0.0.2'
-  spec.add_runtime_dependency 'dry-monads', '>= 0.0.1'
+  spec.add_runtime_dependency 'dry-core', '>= 0.1.0'
 
   spec.add_development_dependency "bundler", "~> 1.6"
   spec.add_development_dependency "rake", "~> 11.0"
   spec.add_development_dependency "rspec", "~> 3.3"
+  spec.add_development_dependency 'dry-monads', '>= 0.1.1'
```

Comment:
not sure actually, there was an issue with 0.1. Anyway, I just released 0.2, so I'll bump the dep with `~> 0.2`